corehq/apps/smsbillables/models.py

```diff
@@ -272,6 +273,14 @@ class Meta:
 
     @property
     def gateway_charge(self):
+        return self.multipart_count * self._single_gateway_charge
```

Comment:
ok, i think that sounds right - we should probably put a comment in the twilio pr that explains that direct_gateway_fee is expected to be the fee of the total sum of all parts of a multipart message